nixops/backends/__init__.py

```diff
@@ -56,6 +71,8 @@ class MachineState(nixops.resources.ResourceState):
     )
     ssh_pinged: bool = nixops.util.attr_property("sshPinged", False, bool)
     ssh_port: int = nixops.util.attr_property("targetPort", 22, int)
+    ssh_user: str = nixops.util.attr_property("targetUser", "root", str)
+    ssh_options: List[str] = nixops.util.attr_property("sshOptions", [], "json")
     public_vpn_key: Optional[str] = nixops.util.attr_property("publicVpnKey", None)
     keys: Mapping[str, str] = nixops.util.attr_property("keys", {}, "json")
     owners: List[str] = nixops.util.attr_property("owners", [], "json")
```

Comment:
we would store the ssh_user in the statefile, and it would always be the same since it is a hard-coded string in the expr